manifests/generated/rbac-kubevirt.authorization.k8s.yaml.in

```diff
@@ -316,6 +316,30 @@ subjects:
   name: kubevirt-handler
   namespace: {{.Namespace}}
 ---
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: kubevirt-privileged-psp
+spec:
+  allowedHostPaths:
```

Comment:
I wonder, don't we need the same for virt-launcher? It references host paths too.
Also virt-launcher will require some privileged capabilities under some circumstances ...

Reply:
Yes, virt-launcher also need PSP to run. 
virt-laucher is invoked when VMI is started in specified namespace. So proper PSP should be associated with certain role, then create role bindings for specified service account.
I think this should be controlled by user manually when provisioning VM or VMI.

To do this automatically when user provision vmi in specified namespace:
1, create role with proper PSP
2, create role bindings for certain service account
3, start virt-launcher as certain service account  in specified namespace 
When deprovision the VMI, also need to check whether it need to clean role, role bindings in the namespace or not.
Is it necessary to implement such feature?

Reply:
@yu2003w I think that `virt-controller` needs a PSP, so that `virt-controller` can create Pods  which contain the VMI. For virt-handler it should be ok if you bind it to the virt-handler service account, but for virt-controller we need that virt-controller has the privileges and not a potential service account in the users namespace.

Reply:
@rmohr  yes, you're correct. I misunderstood the logic.
I did some basic testing, it seems that Virt-controller needs capabilities NET_ADMIN and SYS_NICE and volumes. 
The detailed psp is as below,
apiVersion: extensions/v1beta1
kind: PodSecurityPolicy
metadata:
  creationTimestamp: 2019-05-28T05:51:59Z
  name: kubevirt-controller-psp
  resourceVersion: "2495603"
  selfLink: /apis/extensions/v1beta1/podsecuritypolicies/kubevirt-controller-psp
  uid: b58a6a9a-810c-11e9-809d-0016ac101123
spec:
  allowPrivilegeEscalation: true
  allowedCapabilities:
  - NET_ADMIN
  - SYS_NICE
  fsGroup:
    rule: RunAsAny
  privileged: true
  runAsUser:
    rule: RunAsAny
  seLinux:
    rule: RunAsAny
  supplementalGroups:
    rule: RunAsAny
  volumes:
  - '*'

It requires less conditions than psp required by virt-handler.
I will prepare code and commit code this week.

Reply:
@yu2003w thanks, very nice :+1: Looking forward to bring this PR in.

Reply:
@rmohr Code checked in.